src/tests.c

```diff
@@ -96,7 +96,7 @@ void random_group_element_test(secp256k1_ge *ge) {
 void random_group_element_jacobian_test(secp256k1_gej *gej, const secp256k1_ge *ge) {
     secp256k1_fe z2, z3;
     do {
-        random_field_element_test(&gej->z);
+        random_fe_test(&gej->z);
```

Comment:
In #1358, the following multiple lines are summarized:

```cpp
  do {
        random_field_element_test(&gej->z);
        if (!secp256k1_fe_is_zero(&gej->z)) {
            break;
        }
    } while(1);
```

Reply:
Yes, in our library I only removed the redundant `random_field_element_test` and used 'random_fe_test'.  Summarising the loop with 'random_fe_non_zero_test' was not my initial intent as I wanted to make minimal change. I'll port it since it is causing confusion.